internal/pkg/table/destination.go

```diff
@@ -918,6 +924,34 @@ func compareByRouterID(path1, path2 *Path) (*Path, error) {
 	}
 }
 
+func compareByNeighborAddress(path1, path2 *Path) *Path {
+	// Select the route received from the peer with the lowest peer address as
+	// per RFC 4271 9.1.2.2. g
+	log.WithFields(log.Fields{
+		"Topic": "Table",
+	}).Debug("enter compareByNeighborAddress")
+
+	p1 := path1.GetSource().Address
+	if p1 == nil {
+		return path1
+	}
+	p2 := path2.GetSource().Address
+	if p2 == nil {
+		return path2
+	}
+
+	for i := 0; i < 16; i = i + 4 {
```

Comment:
This function should handle ipv6 too?
how about simply using bytes.Compare()?

Reply:
updated w/bytes.Compare()